### PR TITLE
Updated Readme.md

### PR DESCRIPTION
**Pull request by Sowham-3098**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,16 +59,47 @@ Facing any issues? Feel free to open a ticket.
...
+- Clone the Repo by going to your local Git Client in a particular local folder in your local machine by using this command with your forked repository link in place of below given link: <br/>
+  `git clone https://github.com/hacktobered/hacktobered.com`
+- Create a branch using below command.
+  `git branch <your branch name>`
+- Checkout to your branch.
+  `git checkout <your branch name>`
+- Add your code in your local machine folder.
+  `git add . `
+- Commit your changes.
+  `git commit -m"<add your message here>"`
```
Comment (ankitwww) [inline README.md]: could you please add a space after -m

File: README.md [hunk 2]
```diff
@@ -59,16 +59,47 @@ Facing any issues? Feel free to open a ticket.
...
+- Drop a :star: on the Github repository (optional)<br/>
+
+- Before Contribute Please read [CONTRIBUTING.md](https://github.com/hacktobered/hacktobered.com/blob/main/CONTRIBUTING.md) and [CODE_OF_CONDUCT.md](https://github.com/hacktobered/hacktobered.com/blob/main/CODE_OF_CONDUCT.md)
+
+- Create an issue of the project or a feature you would like to add in the project and get the task assigned for youself.(Issue can be any bug fixes or any feature you want to add in this project).
+
+- Fork the repo to your Github.<br/>
+
+- Clone the Repo by going to your local Git Client in a particular local folder in your local machine by using this command with your forked repository link in place of below given link: <br/>
+  `git clone https://github.com/hacktobered/hacktobered.com`
```
Comment (ankitwww) [inline README.md]: Should not they clone their fork?

**Comment by ankitwww** [comment]
Thanks for quick response @Sowham-3098 ... got few comments for you - please resolve them whenever you get time

**Comment by Sowham-3098** [comment]
@ankitwww please check if it is all right

**Pull request merged**